unity-renderer/Assets/Scripts/MainScripts/DCL/Components/UI/UIInputText/UIInputText.cs

```diff
@@ -12,7 +12,37 @@ public class UIInputText : UIShape<UIInputTextRefContainer, UIInputText.Model>
         [System.Serializable]
         new public class Model : UIShape.Model
         {
-            public TextShape.Model textModel;
+            public float outlineWidth = 0f;
+            public Color outlineColor = Color.white;
+
+            public Color color = Color.white;
+
+            //These values exist in the SDK but we are doing nothing with these values
+            public float thickness = 1f;
+            public Color placeHolderColor = Color.white;
+            public float maxWidth = 100f;
+            public bool autoStretchWidth = true;
+            public Color background = Color.black;
+            public string fontWeight = "normal";
```

Comment:
Should we remove them as well?

Reply:
Yes, there is unused properties in UIText, UIInputText and TextShape. We should remove them in the SDK or implement the functionality in Unity